### PR TITLE
Add open letter to tech workers in Barcelona

### PR DESCRIPTION
**Pull request by rossf7**

### CODE REVIEW

**Review by jlenis5 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,116 @@
...
+- Firma el [Compromiso climático para trabajadores tecnológicos](https://pledge.climateaction.tech/) de ClimateAction.tech 
+- Unéte a [ClimateAction.tech Slack](https://climateaction.tech/), al canal `#local-spain`.
+
+# Carta oberta a treballadors tecnològics en Barcelona
+
+## Qué està pasant?
+
+Som un grup de treballadors tecnològics a Barcelona que volem participar en el [GlobalClimateStrike](https://globalclimatestrike.net) entre el 20 i el 27 de setembre. Volem mostrar el nostre suport als [Fridays for Future Barcelona](https://twitter.com/f4f_barcelona) i al moviment de vaga escolar. També volem reconèixer el paper exercit per la indústria tecnològica a la crisi climàtica i demostrar que estem compromesos a trobar solucions.
+
+## Per qué?
```
Comment (jlenis5) [inline README.md]: Per què?

File: README.md [hunk 2]
```diff
@@ -1 +1,116 @@
...
+
+El sector tecnològic és una de les indústries que té un major impacte en la crisi climàtica. Els centres de dades que alimenten les aplicacions i els llocs web que fem servir actualment són responsables d'aproximadament el 2% de les emissions globals de gasos d'efecte hivernacle, el que fa que les seves emissions siguin iguals a la indústria de l'aviació i fins i tot es projecta que la superin el 2020 [1](https://www.theguardian.com/environment/2017/dec/11/tsunami-of-data-could-consume-fifth-global-electricity-by-2025) [2](https://earther.gizmodo.com/the-planet-needs-a-new-internet-1837101745).
+
+## A on i quand?
```
Comment (jlenis5) [inline README.md]: On i Quan?

File: README.md [hunk 3]
```diff
@@ -1 +1,116 @@
...
+## Hay reglas?
+
+- Esta será una protesta pacífica, no violenta para mostrar apoyo al movimiento de huelga estudiantil. 
+
+## Cómo puedo participar?
+
+- Sigue a [@TechForFutureBCN](https://twitter.com/tech4futurebcn) en Twitter para saber las novedades. 
+- Inscríbete en la huelga del [27 de Septiembre](https://act.350.org/event/globalclimatestrike/18202)
+- Firma el [Compromiso climático para trabajadores tecnológicos](https://pledge.climateaction.tech/) de ClimateAction.tech 
+- Unéte a [ClimateAction.tech Slack](https://climateaction.tech/), al canal `#local-spain`.
```
Comment (jlenis5) [inline README.md]: Únete a [ClimateAction.tech Slack](https://climateaction.tech/) al canal `#local-spain`.

**Comment by rossf7** [comment]
@jlenis5 Thanks! Changes made.

**Comment by pipo02mix** [comment]
nice!

**Pull request merged**